src/components/DocsSwitcher/docsPicker.module.css

```diff
@@ -0,0 +1,80 @@
+.container {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  padding: 8px 16px;
+  gap: 8px;
+  margin-bottom: 20px;
+  position: relative;
+  background: #f5f7fd;
+  border: 1px solid #dfe7fb;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  cursor: pointer;
+  transition: all 0.2s ease;
+}
+
+.logo {
+  width: 24px;
+  height: 24px;
+}
+
+.caret {
+  width: 16px;
+  height: 16px;
+  background: #ffffff;
+  border: 1px solid #375bd2;
+  border-radius: 50%;
+  position: absolute;
+  right: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.caret span {
+  width: 4px;
+  height: 4px;
+  border-right: 1px solid #375bd2;
+  border-bottom: 1px solid #375bd2;
+  transform: rotate(45deg);
+}
+
+.container:hover {
+  background: #e9f0ff;
+}
+
+.menu {
+  position: absolute;
+  top: 40px;
+  left: 0;
+  width: 200px;
+  background: #ffffff;
+  border: 1px solid #dfe7fb;
+  border-radius: 8px;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
+  z-index: 100;
+  transition: all 0.2s ease;
+}
+
+.menu img {
```

Comment:
spacing incorrect:
- image 20
- gap between image and text 8px
- padding 12px 8px
- outer padding 12px 16px

<img width="546" alt="image" src="https://github.com/thisdot/chainlink-docs/assets/9040770/5ab76f6c-d79f-42b0-af0d-d426135b2b80">